HandsOn/Group56/mappings/zonasVerdes.yml

```diff
@@ -0,0 +1,21 @@
+prefixes:
```

Comment:
General comment: I went to the ontology and I see that you have defined 8 classes, so you have to have, at least, 8 TriplesMap (not one), where you define the rule to generate the subject of each class and its associated Data and Object Properties.  For this second step, predicate object maps definition (po) you have to go Data and Object Properties and see where the domains of these properties are equal to the class of the triplesMap. For example, for district, you will have something like 
```
mappingDistrict:
    sources:
      - [ZonasVerdes-updated.csv~csv]
    s: http://group56.org/zonas-verdes/District/$(Distrito)
    po:
     - [a, ex:District]
     - [ex:nombreDistrito, $(Distrito)]
   (and more)
```

Remember to go to the individual's resource name strategy and see what should be the uris for your subjects (I didn't find it In your folder but I assume that would be something like: http://group56.org/zonas-verdes/District/$(District)